docs/modules/ic-identity-guide/pages/hello-guide.adoc

```diff
@@ -0,0 +1,62 @@
+# Windows Hello Guide for Internet Identity
+
```

Comment:
I'd suggest we add a brief introduction that explains what this guide is for, e.g. as follows: 

> Internet Identity supports Windows Hello as authentication method. This guide explains how to set up Windows Hello authentication for an existing Identity Anchor that was set up either on your phone or using a security key. The setup is explained below in Options A and B, respectively.